### PR TITLE
Make cleanoutserver step mandatory in drain documentation.

### PR DESCRIPTION
**Pull request by neunhoef**
Internal Ref:
- https://github.com/arangodb/planning/issues/3733

### CODE REVIEW

File: docs/Manual/Deployment/Kubernetes/Drain.md [hunk 1]
```diff
@@ -407,7 +421,7 @@ much data is stored in the pod, your mileage may vary, moving a terabyte
 of data can take considerably longer!
 
 If the optional step of
-[cleaning out a DBserver manually](#clean-out-a-dbserver-manually-optional)
+[cleaning out a DBserver manually](#clean-out-a-dbserver-manually)
 has been performed beforehand, the grace period can easily be reduced to 60
 seconds - at least from the perspective of ArangoDB, since the server is already
 cleaned out, so it can be dropped readily and there is still no risk.
```
Comment (Simran-B) [inline docs/Manual/Deployment/Kubernetes/Drain.md]: @neunhoef It still says "optional" here. Not sure if the entire sentence should be rewritten...

Comment (neunhoef) [reply docs/Manual/Deployment/Kubernetes/Drain.md]: It should, hang on...

Comment (neunhoef) [reply docs/Manual/Deployment/Kubernetes/Drain.md]: Done.

**Comment by Simran-B** [comment]
What is this sentence supposed to say?

> (please compare the above output of the `/_admin/cluster/health` API).

`compare above output to the output of ...` ?

Does the format follow the one of the cluster health API, or should the user check if both report the same things?

**Comment by sleto-it** [comment]
@Simran-B to reply to your question:

I believe here we are asking the following:

- we are asking to call /_admin/cluster/cleanOutServer, and we have to pass to this call a json document with the name of the dbserver to be clened-out . In our example {"server":"DBServer0006"}

- "(please compare the above output of the /_admin/cluster/health API)" I believe means the following: we are saying to the user that this "DBServer0006" is not a name we invented - it is the one we get as "ShortName" from the /_admin/cluster/health

I believe this was the meaning - please could you reward and make it more clear?
Does this answer? so we merge and publish

thanks

**Pull request merged**